### DISCUSSION TITLE
GitHub application building roadmap

### DISCUSSION THREAD

**Discussion by eternyle** [community/community#110715] [Programming Help]
### Select Topic Area

Question

### Body

Hi,

I would love to have some resources about building a GitHub paid apps for users.

Where can I find some good resources or road map?

thank you

**Comment by AdRohal** [comment]
I hope that will help you :
https://systango.medium.com/step-by-step-guide-to-create-a-github-app-2c18628325b
https://dev.to/github/building-github-apps-for-fun-and-profit-4mid

**Reply by eternyle** [reply]
Thank you for the help. May I ask you more about this topic? Are you on social or email?

Thank you

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @eternyle ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
